tgui/tgui.html

```diff
@@ -16,7 +16,8 @@
       };
     </script>
     <link rel='stylesheet' href='tgui.css'/>
-    <link rel='stylesheet' href='spritesheet_pipes.css'/>
+    <!-- This is processed in byond, so interfaces can override the html head if needed, for custom sheets of style etc -->
+    (customheadhtml)
```

Comment:
basically just a style thing but I might suggest `<!--customheadhtml-->` for this